shared/types/applicant.ts

```diff
@@ -19,6 +22,7 @@ export const applicantDecisions = [
   'Rejected',
 ] as const;
 export const zApplicantDecision = z.enum(applicantDecisions);
```

Comment:
not sure if we reviewed this together, but might I suggest a decision for someone that we want but we are full? Or have an "Other" option? Having the enum kinda contradicts the generic team db structure